Cargo.toml

```diff
@@ -87,6 +87,7 @@ bincode = { version = "1.3" }
 blake3 = { version = "1.5" }
 bytes = { version = "1.4" }
 chrono = { version = "0.4" }
+config = { version = "0.14", features = ["yaml"] }
```

Comment:
let's only put this in notary server's Cargo.toml since its the only mod using this crate now